### PR TITLE
Implement a8j as abj

### LINKED ISSUE #3421: Implement a8j
By the way, shall it be renamed `ab` instead?

### PR DESCRIPTION
**Pull request by jvoisin**
This should close #3421

### CODE REVIEW

File: libr/anal/p/anal_x86_cs.c [hunk 1]
```diff
@@ -267,6 +267,7 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len) {
 		}
 		switch (insn->id) {
 		case X86_INS_FNOP:
+			op->family = R_ANAL_OP_FAMILY_FPU;
```
Comment (crowell) [inline libr/anal/p/anal_x86_cs.c]: what the f is an FNOP?

Comment (radare) [inline libr/anal/p/anal_x86_cs.c]: > On 07 Oct 2015, at 05:55, Jeffrey Crowell <notifications@github.com> wrote:
> 
> In libr/anal/p/anal_x86_cs.c:
> 
> > @@ -267,6 +267,7 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len) {
> >  		}
> >  		switch (insn->id) {
> >  		case X86_INS_FNOP:
> > +			op->family = R_ANAL_OP_FAMILY_FPU;
> what the f is an FNOP?
> 
I think its "fucking" or "fantasy".

Comment (radare) [inline libr/anal/p/anal_x86_cs.c]: no srsly, this is an FPU nop. and if you read the code you'll discover that it's already set to type=NOP

**Comment by radare** [comment]
Murgid!

**Pull request closed without merging**